### ISSUE TITLE
ADA - Issue 3: Missing Skip navigation link

### ISSUE THREAD

**Issue by artjynx** [TAMULib/tamu-library-components#424]
A link is not present which allows users to skip over navigation or other content. A link that provides functionality for the user to jump over navigation or other elements or jump to the main content of the page greatly assists keyboard users in navigating the web page.
Accessibility guidelines
[2.4.1 Bypass Blocks](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FTR%2FWCAG21%2F%23bypass-blocks&data=05%7C01%7Cartjynx%40library.tamu.edu%7Cabb64fa5c8244c6c447e08da64cbe4ac%7C68f381e346da47b9ba576f322b8f0da1%7C1%7C0%7C637933124123818269%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=y07dVgwXG5NgP7Buux2l05x65g00Kfu3mnOxgy7TrZc%3D&reserved=0) (Level A): A mechanism is available to bypass blocks of content that are repeated on multiple Web pages.
Recommendation
Ensure that the skip link is functioning properly and that the link text adequately describes the link functionality. If the skip link is hidden from sighted users, it should be made visible within the page when it has keyboard focus and must be accessible via the keyboard

**Comment by jcreel** [comment]
Let's implement this by having the tamu-components script introduce the "skip navigation" link to the top of the page when it's brought in.
